src/components/data/SongTable/UpcomingSongList.tsx

```diff
@@ -76,7 +76,7 @@ const QueueHeader = () => {
     <Flex mt={4} gap={2} alignItems="center">
       <Text fontSize={["md", "lg"]}>
         <Text opacity={0.66} as={"span"}>
-          {t("Queue")}
+          {t("Queue") as string}
```

Comment:
react-i18next is still not ready for React 18 (but they claim it is https://github.com/i18next/react-i18next/issues/1476)